src/lib/cache/SubscriptionCache.cpp

```diff
@@ -219,6 +219,30 @@ Subscription* SubscriptionCache::lookupById
 
 /* ****************************************************************************
 *
+* SubscriptionCache::lookupById - 
```

Comment:
Yeah, I commented on that above.
I haven't removed the source code of the old cache yet, so there will be a few nmodifications of this "dead code" in this PR.
Just ignore it, the library will soon be completely removed.